script.js

```diff
@@ -1,55 +1,67 @@
-function Animal(name){
-    this.name = name;
-}
+class Animal {
+    constructor(name) {
+        this.name = name;
+    }
 
-Animal.prototype.action = function(){
-  document.getElementById(this.actionSoundName).play();
-  
-}
+    action(element) {
+        document.getElementById("imgAnimal").style.display="block";
+        document.getElementById(this.actionSoundName).play();
 
-Animal.prototype.putInTheDocument = function(){
-    var petsTable = document.getElementById("petsTable");
-    var petTR = document.createElement("tr");
-  
-    var petNameTD = document.createElement("td");
-    petNameTD.textContent = this.name;
-    petTR.appendChild(petNameTD);
-  
-    var petLegsTD = document.createElement("td");
-    petLegsTD.textContent = this.legs;
-    petTR.appendChild(petLegsTD);
-  
-    var petActionTD = document.createElement("td");
-    var petActionTDButton = document.createElement("button");
-    petActionTDButton.textContent = this.actionText;
-    petActionTD.appendChild(petActionTDButton);
-    petTR.appendChild(petActionTD);
-    
-    petActionTDButton.onclick = this.action.bind(this);
-    petsTable.querySelector("tbody").appendChild(petTR)
-}
+    }
 
+    showImage() {
+        document.getElementById("imgAnimal").src = this.image;
+        console.log(this.image)
+    }
 
-function Cat(name){
-    Animal.call(this, name);
-    this.legs = 4;
-    this.actionText = "Meoow"
-    this.actionSoundName = "meow"
-}
+    putInTheDocument() {
+        
+        var petsTable = document.getElementById("petsTable");
+        let animalsTR = document.createElement("tr");
+
+        var petNameTD = document.createElement("td");
+        petNameTD.textContent = this.name;
+        animalsTR.appendChild(petNameTD);
 
-Cat.prototype = Animal.prototype;
+        var petLegsTD = document.createElement("td");
+        petLegsTD.textContent = this.legs;
+        animalsTR.appendChild(petLegsTD);
 
-function Monkey(name){
-    Animal.call(this, name);
-    this.legs = 2;
-    this.actionText = "Scream";
-    this.actionSoundName = "scream";
+        var petActionTD = document.createElement("td");
+        var petActionTDButton = document.createElement("button");
+        petActionTDButton.textContent = this.actionText;
+        petActionTD.appendChild(petActionTDButton);
+        animalsTR.appendChild(petActionTD);
+
+        animalsTR.onclick = this.showImage.bind(this);
```

Comment:
Buraya yazdığının `tr`tıklandığında image'ı getirmesi lazım ama css dosyanda `display:none` olarak yazdığın için bu kısım düzgün çalışmıyor. button tıklandığında `none`ı `block`a çektiğinde düzeliyo